tests/Functional/Driver/Mysqli/ResultTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\Driver\Mysqli;
+
+use Doctrine\DBAL\Driver\Mysqli\Result;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
+use mysqli;
+use mysqli_driver;
+use mysqli_sql_exception;
+
+use function assert;
+use function sprintf;
+
+use const MYSQLI_REPORT_ERROR;
+use const MYSQLI_REPORT_OFF;
+use const MYSQLI_REPORT_STRICT;
+
+/** @requires extension mysqli */
+final class ResultTest extends FunctionalTestCase
+{
+    private const TABLE_NAME = 'tesult_test_table';
```

Comment:
A test result is a "tesult". But I assume it's a typo.
```suggestion
    private const TABLE_NAME = 'result_test_table';
```

Reply:
It was a typo. thank you! :rofl: